appserver/microprofile/health-glassfish/pom.xml

```diff
@@ -0,0 +1,89 @@
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.glassfish.main.microprofile</groupId>
+        <artifactId>microprofile-parent</artifactId>
+        <version>7.0.21-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>health-glassfish</artifactId>
+    <packaging>glassfish-jar</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.release>17</maven.compiler.release>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.glassfish.main.microprofile</groupId>
+            <artifactId>microprofile-health</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.json.bind</groupId>
+            <artifactId>jakarta.json.bind-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <release>17</release>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <supportedProjectTypes>
+                        <supportedProjectType>glassfish-jar</supportedProjectType>
+                    </supportedProjectTypes>
+                    <instructions>
+                        <!-- A list of packages that are exposed to classes outside the bundle -->
+                        <Export-Package>
+                            org.glassfish.microprofile.impl;
+                        </Export-Package>
+                        <Import-Package>
+                            jakarta.enterprise.context.spi;
+                            jakarta.enterprise.event;
+                            jakarta.enterprise.inject;
+                            jakarta.enterprise.inject.spi;
+                            jakarta.json.bind;
+                            jakarta.servlet;
+                            jakarta.servlet.http;
+                            org.eclipse.microprofile.health;
+                            org.glassfish.internal.api;
+                            org.glassfish.microprofile.health;
+                            org.glassfish.hk2.api;
+                            org.glassfish.hk2.utilities;
+                        </Import-Package>
```

Comment:
The `<Import-Package>` block can be removed. By default, all imported packages are added automatically: https://felix.apache.org/documentation/subprojects/apache-felix-maven-bundle-plugin-bnd.html#_import_package
```suggestion
```